docs/man-openmpi/man7/index.rst

```diff
@@ -0,0 +1,8 @@
+Config file manual pages (section 7)
```

Comment:
This should say "General information", not "Config file", right?

Reply:
good catch!